manifests/istio-telemetry/grafana/dashboards/istio-performance-dashboard.json

```diff
@@ -818,7 +818,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum(container_fs_usage_bytes{job=\"kubernetes-cadvisor\", container=\"istio-proxy\"})",
```

Comment:
this is safe because k8s no longer has dual jobs for cadvisor ?

Reply:
Good point. I tested this to make sure it wasn't double, but failed to consider the original reason why it was doubled in the first place. Ill try to figure that out and report back

Reply:
on gke 1.15 this just shows 1x, so no issue there. I also spun up an old 1.13 gke cluster and same thing, its not duplicated. I don't know how this happened before... the only context I found was https://github.com/istio/installer/pull/209#discussion_r288614802.

There IS duplicate if you don't filter on `container` since there will be one that has no `container` label that is the aggregated metric (?), but that doesn't seem to be the same issue..

So I have no idea really how it was happening before, but it works now on GKE at least. I also looked at a number of public grafana dashboards/prometheus scraping configs and all of them are doing this